editor/src/utils/utils.ts

```diff
@@ -281,6 +281,11 @@ export function generateUUID(): string {
   return UUID().replace(/-/g, '_')
 }
 
+// Returns a UUID v4 with dashes removed.
+export function strippedUUID(): string {
+  return UUID().replace(/-/g, '')
```

Comment:
wouldn't we be better off (for debuggability) to keep the dashes?

Reply:
it would create a discrepancy I don't like with the hashed IDs which have the nice property of being overlappable to v4 (same length) but don't have dashes 🤔

Reply:
hmm, wouldn't it be a useful information to know that the uid was hashed or a UUID? for debugging purposes

Reply:
yeah it's a reasonable point, no issue from my side

Reply:
(although at this point we could even prefix the ids)

Reply:
https://github.com/concrete-utopia/utopia/pull/5986/commits/7058bed0e78debe856e98d038d31800288beb1c3